helm/azuredisk-csi-driver-app/templates/crd-install/crd-network-policy.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.snapshot.enabled }}
+kind: NetworkPolicy
```

Comment:
why do we need a dedicated `netPol` for the `pod/crd-jod`?

As the `pod.spec` doesn't contain anything special (no `hostnetwork=true` or a special `kubeconfig`) it seems that the pod is running with a `serviceAccount` attached to communicate with the k8s API server. For that if i'm not mistaken, i think the netpol isn't required.